docs/Guides/Recommendations.md

```diff
@@ -298,3 +299,19 @@ readinessProbe:
     timeoutSeconds: 3
     successThreshold: 1
     failureThreshold: 5
+```
+
+## Running Multiple Instances
+<a id="multiple"></a>
+
+While [Use A Reverse Proxy](#use-a-reverse-proxy) explains why 
```

Comment:
I don't think it is true. You are perfectly fine to scale horizontally with reverse proxy.

The recommendation of reverse proxy only because they did better in certain area than using `node`.
For example `TLS`, `static file serving`.

Reply:
Got it, I misread the `1. The app needs multiple instances to handle load.` part, I guess. Thanks, will fix.